### PR TITLE
Add BSA Tracking Script

### PR DESCRIPTION
**Pull request by JonathanPorta**
- Allows for realtime ad performance monitoring.

image

### CODE REVIEW

**Comment by bobinson** [comment]
@JonathanPorta  - Can you also look into header bidding ? Enabling header bidding is the best possible means to improve monetization.

**Comment by jredbeard** [comment]
@bobinson we already do this through our current advertising partner (bidding between multiple ad networks)... prior to that we were doing it with GPT and prebid.js

File: src/server/server-html.jsx [hunk 1]
```diff
@@ -212,6 +218,19 @@ export default function ServerHTML({
                 {assets.script.map((href, idx) => (
                     <script key={idx} src={href} />
                 ))}
+                {gptEnabled ? (
+                    <script
```
Comment (roadscape) [inline src/server/server-html.jsx]: async=false?

Comment (JonathanPorta) [reply src/server/server-html.jsx]: The browser loads scripts synchronously unless you specify `async=true`. BSA was telling me not to add that tag.

Info here if you want to read: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script

**Pull request merged**